.github/workflows/stg-from-version-build-push-tag-base-image.yaml

```
@@ -0,0 +1,125 @@
+      - name: Download Agent
+        run: |
+          ./bin/download.sh ${{ github.event.inputs.Tag }}
+
+      - name: Copy Agent Files
+        run: |
+          ./bin/copy.sh ${{ github.event.inputs.Tag }} ${{ github.event.inputs.CopyVersionScript }}
+
+      - name: Build Docker Images
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:1:47: Double quote to prevent globbing and word splitting

(shellcheck)